python/phylanx/ast/transformation.py

```diff
@@ -527,8 +555,11 @@ def recompile(self, a, allowreturn=False):
         "shape": "shape",
         "sqrt": "square_root",
         "transpose": "transpose",
-        "vstack": "vstack"
+        "vstack": "vstack",
         # slicing operations
+        # ranges
+        "range" : "range",
```

Comment:
This is not necessary as `range` and `xrange` are not NumPy methods.

Reply:
thanks - will remove.